### PR TITLE
Add Documentation for CSV Ignore Errors

### PR DESCRIPTION
**Pull request by royi-luo**
Addresses #252

### CODE REVIEW

**Review by ray6080 — APPROVED** [review]
Can u also add this for `LOAD FROM`?

File: src/content/docs/import/csv.md [hunk 1]
```diff
@@ -54,6 +55,42 @@ The following statement will load `user.csv` into User table.
...
+2,2147483650
+```
+
+The following statement will load only the first row of `vPerson.csv` into `Person` table, skipping the malformed second row.
+
+```cypher
+COPY Person FROM "vPerson.csv" (header=false, ignore_errors=true);
+```
+
+The warnings table stores any errors that were skipped during the copy.
```
Comment (prrao87) [inline src/content/docs/import/csv.md:78]: Is this really a table, tho? We're showing below that it's retrieved via a `CALL` clause, so in theory it's treated as a query result, right? We haven't described anywhere above that there's something called a warnings table, so it's important that we use the right terminology.
cc @ray6080

File: src/content/docs/import/csv.md [hunk 2]
```diff
@@ -54,6 +55,42 @@ The following statement will load `user.csv` into User table.
...
+
+```cypher
+CALL show_warnings() RETURN *;
+```
+
+Output:
+```
+┌──────────┬─────────────────────────────────────────────────────────────────────────────┬──────────────────────────────────────────────────────────────────────────────────────────┬─────────────┬────────────────────┐
+│ query_id │ message                                                                     │ file_path                                                                                │ line_number │ reconstructed_line │
```
[the remark this answers is not in the document]
Comment (royi-luo) [reply src/content/docs/import/csv.md:87]: I'll update this in the code. Should I update the docs to `skipped_line` right now or should I wait until the code actually reflects this?

File: src/content/docs/import/csv.md (hunk 1 above)
Comment (royi-luo) [reply src/content/docs/import/csv.md]: Updated, let me know if the new description is good

File: src/content/docs/import/csv.md [hunk 3]
```diff
@@ -54,6 +55,42 @@ The following statement will load `user.csv` into User table.
...
+│ UINT64   │ STRING                                                                      │ STRING                                                                                   │ UINT64      │ STRING             │
+├──────────┼─────────────────────────────────────────────────────────────────────────────┼──────────────────────────────────────────────────────────────────────────────────────────┼─────────────┼────────────────────┤
+│ 1        │ Conversion exception: Cast failed. Could not convert "2147483650" to INT32. │ vPerson.csv                                                                              │ 2           │ 2,2147483650       │
```
[the remark this answers is not in the document]
Comment (royi-luo) [reply src/content/docs/import/csv.md]: This was because I did some manual editing, I've updated it so that the columns have an appropriate width

**Comment by royi-luo** [comment]
> Can u also add this for `LOAD FROM`?

added

File: src/content/docs/import/csv.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (prrao87) [reply src/content/docs/import/csv.md]: Yup, let's keep this open until that PR is merged.

**Review by prrao87 — APPROVED** [review]
LGTM!

**Pull request merged**